core/server/common/src/main/java/alluxio/master/journal/ufs/UfsJournal.java

```diff
@@ -639,6 +639,10 @@ public String toString() {
 
   @Override
   public synchronized void close() {
+    if (mState.get() == State.PRIMARY && mWriter != null) {
+      LOG.info("Closing journal {}, state {} last journal location {}, next sequence number {}",
```

Comment:
sanity check on the null check

Reply:
if (mState.get() == State.PRIMARY && mWriter != null) {